app/services/bulk_ingest_service.rb

```diff
@@ -158,17 +158,22 @@ def files(path:, file_filters: [])
       file_paths.reject! { |x| x.basename.to_s.start_with?(".") }
       file_paths.reject! { |x| ignored_file_names.include?(x.basename.to_s) }
 
-      BulkFilePathConverter.new(file_paths: file_paths).to_a
+      BulkFilePathConverter.new(file_paths: file_paths, resource: resource).to_a
     end
 
     class BulkFilePathConverter
-      attr_reader :file_paths
-      def initialize(file_paths:)
+      attr_reader :file_paths, :resource
+      def initialize(file_paths:, resource:)
```

Comment:
This looks like it expects the parent resource. Should we call it parent_resource?